README.md

```diff
@@ -1,4 +1,3 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/3e979c81-a866-40ad-8131-3d049da83dd6/deploy-status)](https://app.netlify.com/sites/hermes-ah/deploys)
-[![Build Status](https://travis-ci.org/andela/hermes-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/hermes-ah-frontend)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/3e979c81-a866-40ad-8131-3d049da83dd6/deploy-status)](https://app.netlify.com/sites/hermes-ah/deploys) [![Build Status](https://travis-ci.org/andela/hermes-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/hermes-ah-frontend) [![Maintainability](https://api.codeclimate.com/v1/badges/494423869e319c5e138f/maintainability)](https://codeclimate.com/github/andela/hermes-ah-frontend/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/494423869e319c5e138f/test_coverage)](https://codeclimate.com/github/andela/hermes-ah-frontend/test_coverage)
```

Comment:
I noticed you also included a code climate Test Coverage badge. Are using CC test cover in place of coveralls coverage?

Reply:
> I noticed you also included a code climate Test Coverage badge. Are using CC test cover in place of coveralls coverage?

Yes @walsamlee , the task is to 'run codeclimate test coverage' and not coveralls

Reply:
ok.